src/Compositor.cpp

```diff
@@ -2446,36 +2446,20 @@ void CCompositor::scheduleFrameForMonitor(CMonitor* pMonitor, IOutput::scheduleF
 }
 
 PHLWINDOW CCompositor::getWindowByRegex(const std::string& regexp) {
-    if (regexp.starts_with("active"))
-        return m_pLastWindow.lock();
-
-    eFocusWindowMode mode = MODE_CLASS_REGEX;
+    // Trim leading whitespace
+    auto tRegexp = regexp;
+    auto pos     = tRegexp.find_first_not_of(" \t\n\r");
```

Comment:
trim()

Reply:
I thought removing trailing whitespace could potentially change the regex

Reply:
we already do that in other places so it shouldnt matter